src/content/docs/en/guides/deploy/clever-cloud.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Deploy your Astro Site to Clever Cloud
+description: How to deploy your Astro site to the web on Clever Cloud.
+type: deploy
+i18nReady: true
+---
+import ReadMore from '~/components/ReadMore.astro';
+import { Steps } from '@astrojs/starlight/components';
+import { CardGrid, LinkCard } from '@astrojs/starlight/components';
+
```

Comment:
```suggestion
import { Steps } from '@astrojs/starlight/components';
```

Reply:
_Idem_

Reply:
Can't commit suggestion on outdated files, but applied it in the current file.